tests/phpunit/WC_Stripe_Account_Test.php

```
@@ -527,4 +527,113 @@ public function test_reconfigure_webhooks_on_update_with_current_webhooks() {
+				[
+					'adaptive_pricing'           => 'yes',
+					'optimized_checkout_element' => 'yes',
+				],
+				[
+					'adaptive_pricing'           => 'no',
+					'optimized_checkout_element' => 'no',
+				],
+				false,
+			],
+		];
+	}
```

PHPMD (2.15.0)

[error] The method provider_maybe_reconfigure_webhooks_after_adaptive_pricing_enabled is not named in camelCase. (undefined)

(CamelCaseMethodName)